### ISSUE TITLE
Issue with FuzzyCompleter import

### ISSUE THREAD

**Issue by nebbles** [nebbles/gitcommit#34]
From @pa17

```
Traceback (most recent call last):
  File "/usr/local/bin/gitcommit", line 6, in <module>
    from gitcommit.gitcommit import main
  File "/usr/local/lib/python3.7/site-packages/gitcommit/gitcommit.py", line 39, in <module>
    from .completers import TypeCompleter, FooterCompleter
  File "/usr/local/lib/python3.7/site-packages/gitcommit/completers.py", line 1, in <module>
    from prompt_toolkit.completion import (  # pylint: disable=no-name-in-module
ImportError: cannot import name 'FuzzyCompleter' from 'prompt_toolkit.completion' (/usr/local/lib/python3.7/site-packages/prompt_toolkit/completion/__init__.py)
```

**Comment by nebbles** [comment]
This should have been resolved with the update to 3.0 for prompt_toolkit in ed9d7dc

Suggest that @pa17 runs an update with `pip install --upgrade conventional-commit`

**Comment by nebbles** [comment]
Think this issue is resolved
